rinja_derive/src/lib.rs

```diff
@@ -313,17 +313,17 @@ const BUILT_IN_FILTERS: &[&str] = &[
     "join",
     "linebreaks",
     "linebreaksbr",
-    "paragraphbreaks",
     "lower",
     "lowercase",
+    "paragraphbreaks",
     "safe",
     "title",
     "trim",
     "truncate",
     "upper",
     "uppercase",
-    "urlencode",
     "urlencode_strict",
+    "urlencode",
```

Comment:
You failed the sorting for this one. ;)

(0 is smaller than `_` hehe)

Reply:
I would say the same all the time! But somehow the sort function in VS code, geany, etc. always give precedence to the longer word.

Reply:
Wut. Oh well.